lib/crudry_resolver.ex

```diff
@@ -27,7 +27,7 @@ defmodule Crudry.Resolver do
 
         def get_my_schema(%{id: id}, _info) do
           MyContext.get_my_schema(id)
-          |> nil_to_error(fn record -> {:ok, record} end)
+          |> nil_to_error("my_schema", fn record -> {:ok, record} end)
```

Comment:
Change these strings to "My Schema", since this is used for a human readable message